.github/workflows/poc-e2e-flow-test.yml

```diff
@@ -0,0 +1,81 @@
+# Description: This workflow mimic the flow of behaviors when user using Amplify CLI;
+
+name: 'poc-e2e-flow-test'
+
+on: # TODO: need to change the trigger
```

Comment:
https://github.com/aws-amplify/amplify-backend/issues/961

Reply:
As we spoke offline. Let's get this merged with health checks workflow and handle `e2e` PR label before we go to main.

Reply:
Done https://github.com/aws-amplify/amplify-backend/pull/968